### PR TITLE
Disable Search Guard & Support Elasticsearch 6.4

### PR DESCRIPTION
**Pull request by hossainemruz**
Fixes: https://github.com/kubedb/project/issues/290 , https://github.com/kubedb/project/issues/222

### CODE REVIEW

File: hack/docker/elasticsearch/6.4.0/make.sh [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+GOPATH=$(go env GOPATH)
+REPO_ROOT="$GOPATH/src/github.com/kubedb/elasticsearch"
+source "$REPO_ROOT/hack/libbuild/common/kubedb_image.sh"
+
+DOCKER_REGISTRY=${DOCKER_REGISTRY:-kubedb}
+IMG=elasticsearch
+SUFFIX=v1
+DB_VERSION=6.4.0
+TAG="$DB_VERSION-$SUFFIX"
```
Comment (the-redback) [inline hack/docker/elasticsearch/6.4.0/make.sh]: @tamalsaha , Newly added databases should start without `suffix`? or, with `suffix`? ie, `6.4.0` or `6.4.0-v1`?

Comment (tamalsaha) [reply hack/docker/elasticsearch/6.4.0/make.sh]: `Newly added databases should start without suffix?`  Yes

File: hack/docker/elasticsearch/6.4.0/fsloader/run_fsloader.sh [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+sync
+
+if [ "$SEARCHGUARD_DISABLED" == false ]; then
+case "$MODE" in
+  client)
+    # Run sgadmin in client node (with ordinal 0 only)
+    ordinal="${NODE_NAME##*-}"
+    if [ "$ordinal" == "0" ]; then
+      /fsloader/run_sgadmin.sh
+      /fsloader/fsloader run --mount-file "$searchguard"/sgconfig/sg_internal_users.yml \
```
Comment (tamalsaha) [inline hack/docker/elasticsearch/6.4.0/fsloader/run_fsloader.sh]: I think this should be `exec /fsloader/fsloader run ...`

File: hack/docker/elasticsearch/6.4.0/config/elasticsearch.yml [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+discovery:
+  zen:
+    ping.unicast.hosts: ${DISCOVERY_SERVICE}
+    minimum_master_nodes: ${NUMBER_OF_MASTERS}
+
+# ref: https://github.com/floragunncom/search-guard-ssl/issues/43
+xpack.security.enabled: ${XPACK_SECURITY_ENABLED:false}
+
+######## Start Search Guard Configuration ########
+searchguard.disabled: ${SEARCHGUARD_DISABLED:false}
```
Comment (tamalsaha) [inline hack/docker/elasticsearch/6.4.0/config/elasticsearch.yml]: I want to change this to `SEARCHGUARD_ENABLED`

File: hack/docker/elasticsearch/6.4.0/fsloader/run_fsloader.sh (hunk 2 above)
Comment (hossainemruz) [reply hack/docker/elasticsearch/6.4.0/fsloader/run_fsloader.sh]: none of previous script has `exec` here.

File: hack/docker/elasticsearch/6.4.0/config/elasticsearch.yml (hunk 3 above)
Comment (hossainemruz) [reply hack/docker/elasticsearch/6.4.0/config/elasticsearch.yml]: how can we use negated value here?

File: hack/docker/elasticsearch/6.4.0/fsloader/run_fsloader.sh (hunk 2 above)
Comment (hossainemruz) [reply hack/docker/elasticsearch/6.4.0/fsloader/run_fsloader.sh]: added

File: hack/docker/elasticsearch/6.4.0/config/elasticsearch.yml (hunk 3 above)
Comment (hossainemruz) [reply hack/docker/elasticsearch/6.4.0/config/elasticsearch.yml]: keeping `SEARCHGUARD_DISABLED`

**Pull request merged**